### DISCUSSION TITLE
Is rdtclient supposed to keep a folder of the movie name in the install directory?

### DISCUSSION THREAD

**Discussion by MrJoshGit** [rogerfar/rdt-client#477] [General]
I have a windows install on latest version.

rtdclient is finally working after I found out I need to run it as administrator.

I am seeing a copy of the movie folder that is the same as the real debrid folder name as a copy in my rtdclient install directory.  Is this expected?

I see a link created in my symlinks (download) folder and later removed.  I assume this is expected.
I see the link/folder (renamed via Radarr) created in my media/movies folder.

But I also see a folder created in the rtdclient install directory.

I have added my full debug log here.
[rdtclient.log](https://github.com/rogerfar/rdt-client/files/15097393/rdtclient.log)

![Capture](https://github.com/rogerfar/rdt-client/assets/167652979/6e058ed7-baac-47d0-8276-92c0ab5f3490)

**Comment by MrJoshGit** [comment]
Found this:
https://github.com/rogerfar/rdt-client/issues/438

However, I'm not sure where to change the .db file location to.  Tried the install directory but problem is still persisting.  Anyone have any ideas?

**Comment by rogerfar** [comment]
You change the paths in the appsettings file. But you should setup the download paths in the general settings page.

**Reply by MrJoshGit** [reply]
I changed path to DB file in the appsettings.  Tried different locations.  I have also set the download path in the settings page.  However still am getting duplicate folder structure in the main rdtclient path folder as mentioned above

**Reply by MrJoshGit** [reply]
As a followup, I do see the folder created in my download location as well as the main rdtclient section

**Comment by rogerfar** [comment]
Can you show your settings and appconfig?

**Comment by MrJoshGit** [comment]
Currently using two \ but I have tried with one and it didn't work.  I have also tried moving database path to "C:\\rdtclient\\db\\rdtclient.db"

{
  "Logging": {
    "File": {
      "Path": "C:\\rdtclient\\db\\rdtclient.log",
      "FileSizeLimitBytes": 5242880,
      "MaxRollingFiles": 5
    }
  },
  "Database": {
    "Path": "C:\\rdtclient\\rdtclient.db"
  },
  "Port": "6500",
  "BasePath": null
}

![Screenshot 2024-05-07 121710](https://github.com/rogerfar/rdt-client/assets/167652979/fa143a39-9282-4965-b0cb-8731b4645f90)

**Comment by rogerfar** [comment]
It doesn't make any sense though, the logs indicate `c:\symlinks\radarr\...` and nowhere it's referencing your install directory. I've also never seen this behaviour yet.

If this still happens, what exact time does the folder get create (hh:mm:ss, and ms if available), maybe we can cross reference it with the logs.
